druid/src/main/java/org/apache/calcite/adapter/druid/DruidRules.java

```diff
@@ -362,6 +362,13 @@ private static boolean validAggregate(Aggregate aggregate, DruidQuery query) {
       for (AggregateCall aggCall : aggregate.getAggCallList()) {
         builder.addAll(aggCall.getArgList());
       }
+      for (int index : aggregate.getGroupSet()) {
```

Comment:
@jcamachor i have run the tests and got the same results after/before patch 
` 9 failed, 2 ignored, 46 passed`